python/pyspark/ml/tests.py

```diff
@@ -1473,11 +1473,59 @@ def test_logistic_regression_summary(self):
         self.assertTrue(isinstance(s.fMeasureByThreshold, DataFrame))
         self.assertTrue(isinstance(s.precisionByThreshold, DataFrame))
         self.assertTrue(isinstance(s.recallByThreshold, DataFrame))
+
+        self.assertAlmostEqual(s.accuracy, 1.0, 2)
```

Comment:
care to add these to the scala unit test for binary summary as well?

Reply:
also nit, but should probably add tests for all the new attributes, like `falsePositiveRateByLabel` as below.